pkg/interface/src/views/apps/profile/components/lib/BackgroundPicker.tsx

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
-import { Box, InputLabel, Radio, Input } from '@tlon/indigo-react';
+import React from "react";
+import {
+  Box,
+  Label,
+  ManagedRadioButtonField as Radio,
+  ManagedTextInputField as Input,
+} from "@tlon/indigo-react";
 
-import GlobalApi from '~/logic/api/global';
-import { S3State } from '~/types';
-import { ImageInput } from '~/views/components/ImageInput';
+import GlobalApi from "~/logic/api/global";
+import { S3State } from "~/types";
+import { ImageInput } from "~/views/components/ImageInput";
```

Comment:
I believe our linter for pkg/interface uses single quotes; we should avoid committing this stuff back and forth in future.